.github/workflows/ci.yaml

```diff
@@ -1,45 +1,45 @@
-# yamllint disable rule:line-length
-# yamllint disable rule:braces
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow
 
 name: CI
 
 on:
-  pull_request:
+  pull_request: null
   push:
     branches:
-      - master
```

Comment:
No more master branch.

Reply:
I'd rather keep `master` here because I use it as a template elsewhere.

Reply:
Those thing live in a repo like https://github.com/sanmai/.github
And used from there!

Reply:
Someday, for sure!